src/views/ConfigEditor.tsx

```diff
@@ -88,17 +113,57 @@ const ConfigEditor = (props: ConfigEditorProps) => {
       <Divider />
 
       <ConfigSection title="Connection">
-        <Field label="Access Token">
-          <SecretInput
-            placeholder="GitHub Personal Access Token"
-            data-testid={components.ConfigEditor.AccessToken}
-            value={secureSettings.accessToken || ''}
-            isConfigured={secureJsonFields!['accessToken']}
-            onChange={onSettingUpdate('accessToken', false)}
-            onReset={onSettingReset('accessToken')}
-            width={WIDTH_LONG}
-          />
-        </Field>
+        <RadioButtonGroup
+          options={authOptions}
+          value={selectedAuthType}
+          onChange={onAuthChange}
+          className={styles.radioButton}
+        />
+
+        {selectedAuthType === 'personal-access-token' && (
+          <Field label="Personal Access Token">
+            <SecretInput
+              placeholder="Personal Access Token"
+              data-testid={components.ConfigEditor.AccessToken}
+              value={secureSettings.accessToken || ''}
+              isConfigured={secureJsonFields!['accessToken']}
+              onChange={onSettingUpdate('accessToken', false)}
+              onReset={onSettingReset('accessToken')}
+              width={WIDTH_LONG}
+            />
+          </Field>
+        )}
+
+        {selectedAuthType === 'github-app' && (
+          <>
+            <Field label="App ID">
+              <Input
+                placeholder="App ID"
+                value={jsonData.appId}
+                onChange={onUpdateDatasourceJsonDataOption(props, 'appId')}
+                width={WIDTH_LONG}
+              />
+            </Field>
+            <Field label="Installation ID">
+              <Input
+                placeholder="Installation ID"
```

Comment:
It is quite hard to get to Installation ID, is there a way  to use Client ID?

Reply:
marking as resolved from out discussion in meets